rom/dev/src/flow/cold_reset/fw_processor.rs

```diff
@@ -122,39 +121,47 @@ impl FirmwareProcessor {
     /// handle_fatal_error is called. This prevents a race condition where the SoC
```

Comment:
Make it clear in this documentation that the returned transaction handle is for the FIRMWARE_LOAD command.